scripts/imgr

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Define usage here for faster return
+usage()
+{
+cat << EOF
+usage: imgr <IOCNAME> [--hutch HUTCH] <OPTION>
+
+Control status of all IOCs running in a particular hutch from the command line.
+Current hutch is used if not provided.
+
+List of options:
+imgr IOCNAME [--hutch HUTCH] --reboot soft
+imgr IOCNAME [--hutch HUTCH] --reboot hard
+imgr IOCNAME [--hutch HUTCH] --enable
+imgr IOCNAME [--hutch HUTCH] --disable
+imgr IOCNAME [--hutch HUTCH] --upgrade RELEASE_DIR
+imgr IOCNAME [--hutch HUTCH] --move HOST
+imgr IOCNAME [--hutch HUTCH] --move HOST:PORT
+imgr [--hutch HUTCH] --list [--host HOST] [--enabled_only|--disabled_only]
+ 
+EOF
+}
+if [ $# -lt 1 ]; then
+    echo 'Missing required arguments' >&2
+    usage
+    exit 1
+elif [[ ($1 == "--help") || ($1 == "-h") ]]; then
+    usage
+    exit 0
+fi
+
+# Setup environment
+if [ -x /etc/pathinit ]; then
+    source /etc/pathinit
+else
+    export PSPKG_ROOT=/cds/group/pcds/pkg_mgr
+    export PYPS_ROOT=/cds/group/pcds/pyps
+    export IOC_ROOT=/cds/group/pcds/epics/ioc
+    export CAMRECORD_ROOT=/cds/group/pcds/controls/camrecord
+    export IOC_DATA=/cds/data/iocData
+    export IOC_COMMON=/cds/data/iocCommon
+fi
+PSPKG_OS=$(${PSPKG_ROOT}/etc/pspkg_os.sh)
+export PSPKG_OS
+if [ "$PSPKG_OS" == rhel5 ]; then
+    echo "IocManager 2.0.0 and higher does not run on RHEL5!" >&2
+    exit 1
+fi
+export QT_XKB_CONFIG_ROOT=/usr/share/X11/xkb
+export PSPKG_RELEASE=controls-0.1.9
+source $PSPKG_ROOT/etc/set_env.sh
+DIR=$(readlink -f "$(dirname "${BASH_SOURCE[0]}")")
+PATH=$PATH:$DIR
+
+# Search for hutch in args
```

Comment:
Is it correct to set up the environment before processing args? Minor point I guess, setting up the environment is probably pretty quick here and the script is pretty small. In a larger script this might be an issue.